data/sortOptions.json

```diff
@@ -0,0 +1,53 @@
+[
+    {
+        "id": "792fe292-fc9c-11ec-b939-0242ac120002",
+        "name": "Popularity",
```

Comment:
We can simply call these stars, `Popularity` is not a quantitative measure, and may not correspond to stars directly.

Reply:
I have followed your suggestions and modified the code please review it again now